packages/flutter/lib/src/widgets/semantics_debugger.dart

```diff
@@ -76,6 +76,8 @@ class _SemanticsDebuggerState extends State<SemanticsDebugger> with WidgetsBindi
         });
       }
     });
+    // addPostFrameCallback does *not* schedule a frame.
+    SchedulerBinding.instance.scheduleFrame();
```

Comment:
setState called in a postFrameCallback will not actually schedule a frame. setState only schedules frames when called during SchedulerPhase "idle" and in a postFrameCallback the SchedulerPhase is "postFrameCallback". That's why we have to force a frame in this specific case after calling setState.

I added an explanation and a warning to not copy this pattern in a comment. PTAL